JS_lesson4/Task3.js

```diff
@@ -0,0 +1,26 @@
+// Преобразовать Task 2* таким образом, чтобы age принимался через prompt в привязанной вёрстке, а результат выводился в alert
+const age = prompt(`Введи возраст`);
+const age_2 = 18;
+const age_3 = 60;
+if (isNaN(age)) {
+    alert('Parameter is not a number!');
+    throw new Error('Parameter is not a number!'); 
+}
+switch (true) {
```

Comment:
Seems like you need to convert age to number (cause prompt returns string) just bedore switch statement:
age = +age

Cause now you are using string value comparing with numbers